app/rdd2/CMakeLists.txt

```diff
@@ -142,4 +142,8 @@ target_include_directories(app SYSTEM BEFORE PRIVATE
   ${CMAKE_BINARY_DIR}
   )
 
+if(CONFIG_BOARD STREQUAL "vmu_rt1170")
+  zephyr_linker_sources(ITCM_SECTION boards/vmu_rt1170_itcm_linker.ld)
```

Comment:
Is this equivalent to using `zephyr_code_relocate()`? I'm thinking that would be easier to maintain, but even if we'd like finer-grain control, we could reduce the manual mapping list to those exceptions. Reference: https://github.com/CogniPilot/zephyr/blob/c80dc7a557a2f6bc593d3d8c29239ab24b36443c/samples/sensor/udp_streaming/CMakeLists.txt#L11-L26

Reply:
I see - Keep in mind that this would need to change on every manifest update (e.g: zephyr or nxp-hal code is updated), that otherwise would be contained in a module: so that'd impose additional overhead on the sync-cycle.

Do you have an idea of how much this manual approach saves? I'm good either way but I'm curious how much the overhead is.

Reply:
For our future selves: It'd be useful if you added the steps to generate the list from the SWO trace.